src/Cms/System.php

```diff
@@ -231,18 +205,15 @@ public function init()
 	 * On a public server the panel.install
 	 * option must be explicitly set to true
 	 * to get the installer up and running.
-	 *
-	 * @return bool
 	 */
 	public function isInstallable(): bool
 	{
-		return $this->isLocal() === true || $this->app->option('panel.install', false) === true;
+		return $this->isLocal() === true ||
+			   $this->app->option('panel.install', false) === true;
```

Comment:
Line 212 has mixed tabs and spaces. We could fix this by using spaces after the indentation, but that gets hard to manage. What about this:

```suggestion
		return (
			$this->isLocal() === true ||
			$this->app->option('panel.install', false) === true
		);
```

Reply:
Don't we have that in quite a few places to align items if e.g. half a tab is needed?

Reply:
What is "half a tab"? That would require everyone to use the same tab width. You can see it in the GitHub diff preview above that it breaks once a different tab width is used (here: 8 spaces).

Reply:
To answer your question: I don't know if we use it anywhere else already.

Reply:
We have it a lot: if you do a global search in PHP files for one tab followed by two spaces, you'll see many of the places where we do this. 

> That would require everyone to use the same tab width.
That's true. Then again we have `.editorconfig` defining this for everybody (just GitHub ignoring it, which many consider a bug).

Reply:
I've removed them in all the places and used a pattern that we already have in other places:

```php
return
	$this->isLocal() === true ||
	$this->app->option('panel.install', false) === true;
```